include/pfs/types.hpp

```diff
@@ -620,6 +620,16 @@ struct net_route
     unsigned int irtt;
 };
 
+struct net_arp
+{
+    std::string ipaddr;
+    unsigned int type;
+    unsigned int flags;
+    std::string mac;
```

Comment:
The library tries to use names that are as close as possible to the actual output from procfs, can we use here `hw_address`?

Reply:
Done